### PR TITLE
Fix default value and key for client_url

### LINKED ISSUE #7: Bug: client_service_name requires the protocol
The recent update for issue #5 introduced a bug in that the value of `$settings[client_name]` passed to `phpCAS::client()` requires a protocol (ie. `https`), and the coded default does not include one.

Also, as the relevant `phpCAS::client()` argument name is `$service_base_url`, refactor our variable name to more closely match.

### PR DESCRIPTION
**Pull request by pbarabe**
Updates the default value passed to phpCAS::client() service URL arg to include the protocol.

closes #7

### CODE REVIEW

**Review by ratish-ua — APPROVED** [review]
Looks good @pbarabe

File: README.md [hunk 1]
```diff
@@ -48,36 +49,45 @@ $this->loadComponent(
...
+### Note about parameter key changes
+
+Prior to release 2.0.0, several parameter used different keys.
+Release 2.0.0 updates `apereo/phpcas` to use at least version 1.6, which contains breaking changes.
+For better clarity, the previous parameter key names have been re-mapped to the new names, which
+match variable names as used in the `apereo/phpcas`
+[example client usage](https://github.com/apereo/phpCAS/blob/master/docs/examples/example_simple.php).
+
+- `hostname` changed to `cas_host`
+- `port` changed to `cas_port
```
Comment (ratish-ua) [inline README.md:85]:
```suggestion
- `port` changed to `cas_port`
```

**Review by ratish-ua — APPROVED** [review]
@pbarabe Added a suggestion about missing '`', but everything else looks good in README.md

File: README.md (hunk 1 above)
Comment (pbarabe) [reply README.md:85]: Thanks!

**Pull request merged**